CONTRIBUTING.md

```diff
@@ -1,3 +1,41 @@
+## Guidelines
```

Comment:
```suggestion
## Contribution Guidelines
```

Reply:
this is in case we change `CONTRIBUTING` title

Reply:
what about Contributing Guidelines?

Reply:
@kapunahelewong which title do think is better?

Reply:
Maybe let's just go with a tried title like React:
<img width="660" alt="image" src="https://user-images.githubusercontent.com/6682981/212023308-02803f48-65de-4538-b944-a7f1e8515385.png">

So: `contributing to qwik`